src/main/java/reposense/model/FileType.java

```diff
@@ -17,37 +17,53 @@
  * Represents a file type for use in {@link FileTypeManager}.
  */
 public class FileType {
-    private static final String FILE_TYPE_VALIDATION_REGEX = "[A-Za-z0-9]+";
-    private static final String MESSAGE_ILLEGAL_FILE_TYPE = "The provided file type, %s, contains illegal characters.";
+    private static final String FILE_FORMAT_VALIDATION_REGEX = "[A-Za-z0-9]+";
+    private static final String MESSAGE_ILLEGAL_FILE_FORMAT = "The provided file format, %s, contains illegal "
+            + "characters.";
 
     private String label;
     private List<String> paths;
     private PathMatcher pathsGlob;
 
     public FileType(String label, List<String> paths) {
-        validateFileType(label);
+        validateFileTypeLabel(label);
         this.label = label;
         setPaths(paths);
     }
 
     /**
      * Ensures that the string {@code label} is a valid file type.
-     * @throws IllegalArgumentException if {@code label} does not meet the criteria.
+     * @throws IllegalArgumentException if {@code label} is an empty string.
      */
-    public static void validateFileType(String label) {
-        if (!label.matches(FILE_TYPE_VALIDATION_REGEX)) {
-            throw new IllegalArgumentException(String.format(MESSAGE_ILLEGAL_FILE_TYPE, label));
+    public static void validateFileTypeLabel(String label) {
+        if (label.isEmpty()) {
+            throw new IllegalArgumentException();
```

Comment:
Move the exception message here.

See next comment.

Reply:
Doing so will require the modification of `validateFileTypeLabel` and `FileType` constructor to also take in the full file name just to use the exception message to inform the user that we can't deduce the format **of this particular file**.

Unless if we don't inform the user in the first place and just set the fileType of the file as `other`?